spec/spec_helper.rb

```diff
@@ -1,17 +1,23 @@
 # use . in include path
 $LOAD_PATH.unshift File.dirname(__FILE__) + '/../.'
-require 'json'
-require 'pp'
-require 'dotenv'
-require 'gmail'
+
 require 'capybara/rspec'
 require 'capybara/poltergeist'
+require 'capybara-screenshot/rspec'
```

Comment:
Looks like we don't actually use most of the things we require here. Thoughts on pulling this out and replacing it with something like `bundler.require(:default)` since we're doing a lot of `requires`?

Reply:
debugging remnants that can be removed. Although capy-screenshot may be interesting down the road so we can better understand how a test failed (ie. post the image to slack on fail).